### PR TITLE
Fix TOS link

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
Fixes a broken link in Tagalog translation.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "@bugsnag/plugin-react": "^7.17.0",
     "@floating-ui/react": "^0.19.2",
     "@opentripplanner/base-map": "^3.0.16",
-    "@opentripplanner/core-utils": "^11.2.3",
+    "@opentripplanner/core-utils": "11.3.0",
```
Comment (binh-dam-ibigroup) [inline package.json:46]: Any reason to pin the core-utils version?
```suggestion
   "@opentripplanner/core-utils": "^11.3.0",
```

File: yarn.lock [hunk 2]
```diff
@@ -2387,7 +2387,25 @@
...
+    "@turf/along" "^6.0.1"
+    chroma-js "^2.4.2"
+    date-fns "^2.28.0"
+    date-fns-tz "^1.2.2"
+    graphql "^16.6.0"
+    lodash.clonedeep "^4.5.0"
+    lodash.isequal "^4.5.0"
+    qs "^6.9.1"
+
+"@opentripplanner/core-utils@^11.0.2", "@opentripplanner/core-utils@^11.1.3":
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2408]: Can these core-utils be deduplicated?

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Can you unpin the core-utils version and try deduplicating?

**Comment by amy-corson-ibigroup** [comment]
Core utils was updated by https://github.com/opentripplanner/otp-react-redux/pull/1148 so this is just correcting the Tagalog link now!

**Review by binh-dam-ibigroup — APPROVED** [review]

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Oops! Good fix

**Pull request merged**